<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataParseDialog</class>
 <widget class="QDialog" name="DataParseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="rawpathLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Renishaw txt file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rawpathPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Renishaw info pck file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infopathLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="infopathPushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savepathLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="savepathPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>save spectra folder</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="sampleinfoTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Sample Table (tab-deliminated)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>column names for
sample_no,xrng_0,xrng_1,yrng_0,yrng_1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infokeysLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sample_no,xrng_0,xrng_1,yrng_0,yrng_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="getinfoPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Raman File Info
(read pck if exists)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matchPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Raman File to
Select Sample Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extractPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract Raman
Spectra to File</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sampleComboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="plotComboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>sample (# spectra)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>plot style</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_xy">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="infopathnewPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>new</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copymatchPushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OR copy
map file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readfolderPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Save
Spectra Folder</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_xy2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="avePushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Ave Spectrum
 for each sample</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ramaninfoTextBrowser">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>151</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
